test/src/edu/stanford/nlp/process/WhitespaceTokenizerTest.java

```diff
@@ -61,5 +76,37 @@ public void testCoreLabelTokenizer() {
               (factory, true),
             TEST, RESULTS_EOL);
   }
+  
+  /* Tests the "whitespace only" bound */
```

Comment:
oh, I see what happened.  I had renamed the test "testNoTextWhiteSpaceTokenizer"